cli/flox/src/utils/errors.rs

```diff
@@ -505,12 +507,23 @@ pub fn format_remote_error(err: &RemoteEnvironmentError) -> String {
             Please ensure that you have write permissions to FLOX_CACHE_DIR/remote.
         "},
 
+        RemoteEnvironmentError::ResetManagedEnvironment(ManagedEnvironmentError::FetchUpdates(
+            GitRemoteCommandError::RefNotFound(_),
+        ))
+        | RemoteEnvironmentError::GetLatestVersion(FloxMetaError::CloneBranch(
+            GitRemoteCommandError::AccessDenied,
+        ))
+        | RemoteEnvironmentError::GetLatestVersion(FloxMetaError::CloneBranch(
+            GitRemoteCommandError::RefNotFound(_),
+        )) => formatdoc! {"
+            Environment not found in FloxHub.
```

Comment:
"in" or "on"?

Reply:
"in" according to spec: https://github.com/flox/product/issues/656